kubernetes/crypto_detector/stats/README.md

```diff
@@ -0,0 +1,19 @@
+# Cryptomining Detection System (CDS) Latency Performance Script
+This script measures the latency introduced by the CDS. To execute the script type:
+```
+./start-script.sh n
+```
+Change n with the number of seconds that the script will be collecting statistics.
+
+To delete the scenario after being deployed:
+```
+./clean-env.sh
+```
+
+### Important notes:
+- The script creates a local cluster with Zookeeper and Kafka in Step 1. Comment these lines if not needed
```

Comment:
Note that Helm is required in this step.

Reply:
Included a list of requirements to clarify